src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -166,6 +166,14 @@ const DISPLAY_FEEDBACK_QUESTION_NUMSCALE_INTERVALINVALID =
         'Please enter valid options. The interval is not divisible by the specified increment.';
 const DISPLAY_FEEDBACK_SESSION_VISIBLE_DATEINVALID = 'Feedback session visible date must not be empty';
 const DISPLAY_FEEDBACK_SESSION_PUBLISH_DATEINVALID = 'Feedback session publish date must not be empty';
+const FEEDBACK_QUESTION_CONSTSUMPOINTS_OPTION =
+    "Respondents will have to distribute the total points specified here among the options, "
```

Comment:
The Travis build failed due to `npm run lint` failing. You can check out the Travis log; alternatively, checkout the [process documentation](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-3-fix-the-issue) and take a look at the portion on static analysis.